src/components/common/EthHashInfo/styles.module.css

```diff
@@ -2,11 +2,6 @@
   display: flex;
   align-items: center;
   gap: 0.5em;
-}
-
-.container,
-.address {
-  font-size: inherit;
```

Comment:
This doesn't inherit the `font-size` anymore.

Reply:
The approach from this PR reduces the font-size of the address and prefix form 16px to 14px so even the longest `<short_name>:<address>` can be contained inside the Modal's 600px